src/Core/Form/IdentifiableObject/DataFormatter/BulkCombinationFormDataFormatter.php

```diff
@@ -84,6 +85,13 @@ public function format(array $formData): array
             }
         }
 
+        if (!empty($formData['images']['disabling_switch_images']) && empty($formData['images']['images'])) {
```

Comment:
Just to test, now that the provider provides empty data as `[]` did you check if you still had the same issue when nothing is checked? It's possible that the form data is now correctly filled and you wouldn't need this custom check

It's worth testing but if it didn't change then this is perfectly fine!

Reply:
nope, not working.